tests/rt_utils.sh

```diff
@@ -324,65 +230,44 @@ check_results() {
 
   echo                                                       >  ${REGRESSIONTEST_LOG}
   echo "baseline dir = ${RTPWD}/${CNTL_DIR}"                 >> ${REGRESSIONTEST_LOG}
-  echo "mediator baseline dir = ${RTPWD}/${CNTLMED_DIR}"     >> ${REGRESSIONTEST_LOG}
   echo "working dir  = ${RUNDIR}"                            >> ${REGRESSIONTEST_LOG}
   echo "Checking test ${TEST_NR} ${TEST_NAME} results ...."  >> ${REGRESSIONTEST_LOG}
   echo
   echo "baseline dir = ${RTPWD}/${CNTL_DIR}"
-  echo "mediator baseline dir = ${RTPWD}/${CNTLMED_DIR}"
   echo "working dir  = ${RUNDIR}"
   echo "Checking test ${TEST_NR} ${TEST_NAME} results ...."
 
   if [[ ${CREATE_BASELINE} = false ]]; then
     #
-    # --- regression test comparison ----
+    # --- regression test comparison
     #
     for i in ${LIST_FILES} ; do
       printf %s " Comparing " $i " ....." >> ${REGRESSIONTEST_LOG}
       printf %s " Comparing " $i " ....."
 
-      crst=''
-      if [[ $i =~ RESTART/ ]]; then
-        crst=RESTART/$(basename $i)
-      fi
-
       if [[ ! -f ${RUNDIR}/$i ]] ; then
 
         echo ".......MISSING file" >> ${REGRESSIONTEST_LOG}
         echo ".......MISSING file"
         test_status='FAIL'
 
-      elif [[ ! -f ${RTPWD}/${CNTL_DIR}/$i && ! -f ${RTPWD}/${CNTLMED_DIR}/$i && ! -f ${RTPWD}/${CNTL_DIR}/$crst ]] ; then
+      elif [[ ! -f ${RTPWD}/${CNTL_DIR}/$i ]] ; then
 
         echo ".......MISSING baseline" >> ${REGRESSIONTEST_LOG}
         echo ".......MISSING baseline"
         test_status='FAIL'
 
-      elif [[ ( $COMPILER == "gnu" || $COMPILER == "pgi" ) && $i == "RESTART/fv_core.res.nc" ]] ; then
+      elif [[ $COMPILER == "gnu" && $i == "RESTART/fv_core.res.nc" ]] ; then
 
         # Although identical in ncdiff, RESTART/fv_core.res.nc differs in byte 469, line 3,
         # for the fv3_control_32bit test between each run (without changing the source code)
-        # for GNU and PGI compilers - skip comparison.
-        echo ".......SKIP for gnu/pgi compilers" >> ${REGRESSIONTEST_LOG}
-        echo ".......SKIP for gnu/pgi compilers"
-
-      elif [[ $COMPILER == "pgi"  && ( $i == "RESTART/fv_BC_sw.res.nest02.nc" || $i == "RESTART/fv_BC_ne.res.nest02.nc" ) ]] ; then
-
-        # Although identical in ncdiff, RESTART/fv_BC_sw.res.nest02.nc differs in byte 6897, line 17
-        # (similar for fv_BC_ne.res.nest02.nc) for the fv3_stretched_nest test between each run
-        # (without changing the source code) for the PGI compiler - skip comparison.
-        echo ".......SKIP for pgi compiler" >> ${REGRESSIONTEST_LOG}
-        echo ".......SKIP for pgi compiler"
+        # for GNU compilers - skip comparison.
+        echo ".......SKIP for gnu compilers" >> ${REGRESSIONTEST_LOG}
+        echo ".......SKIP for gnu compilers"
 
       else
 
-        if [[ $i =~ ufs.s2s ]]; then
-          d=$( cmp ${RTPWD}/${CNTLMED_DIR}/$i ${RUNDIR}/$i | wc -l )
```

Comment:
Can we leave this file and add extra netcdf file comparison using nccmp, cprnc or compare_ncfile.py (need to check efficiency) when cmp comparison shows difference?

Reply:
ufs.s2s.* is still being compared because it is in LIST_FILES of tests/tests/<test-name>.  It's just that CNTLMED_DIR no longer exists.

Let me look into these other comparison methods. Where can I find info on compare_ncfile.py?